lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -286,6 +287,27 @@ class ReportUIFeatures {
     this._dom.resetTemplates();
   }
 
+  /**
+   * Jumps to respective audit on page load if hash is provided in URL
+   * Collapsed content is opened automatically
+   * Element is highlighted via CSS selector
+   */
+  _jumpToAudit() {
+    const hash = window.location.hash;
+    if (hash) {
+      const audit = document.querySelector(hash);
+      if (audit) {
+        if (audit.firstElementChild) {
```

Comment:
should probably check that this is a `<details>` (or should we just open any `details` children)